alephium/configuration.ts

```diff
@@ -0,0 +1,61 @@
+import { Configuration } from "./lib/deployment"
+
+const configuration: Configuration = {
+  sourcePath: "contracts",
+  artifactPath: "artifacts",
+
+  networks: {
+    "devnet": {
+      nodeUrl: "http://localhost:22973",
+      mnemonic: "vault alarm sad mass witness property virus style good flower rice alpha viable evidence run glare pretty scout evil judge enroll refuse another lava",
+      deploymentFile: ".deployments.json",
+      scripts: [
```

Comment:
How does the DevX compare to by default running all scripts in "script" directory in the alphabetic order? Hardhat deploy seems to behave this way, which makes some to adopt the following naming convention for deployment scripts: "00_xxx.js", "01_yyy.js", etc. 

e.g. https://github.com/smartcontractkit/hardhat-starter-kit/tree/main/deploy

Reply:
I realized that the scripts are different between devnet and mainnet, which is a good reason to specify all the scripts here. but this question still holds: https://github.com/alephium/wormhole-fork/pull/27/files#r963145258

Reply:
We can also use the same way as hardhat/truffle, because the `DeployFunction` have a `NetworkType` parameter, users can decide whether to deploy the contract according to the network type, which way do you prefer?

Reply:
Sounds great 👍 , will update.